Gems/ScriptCanvas/Code/Editor/Framework/ScriptCanvasReporter.h

```diff
@@ -190,6 +194,7 @@ namespace ScriptCanvasEditor
         bool m_isParseAttemptMade = false;
         bool m_isReportFinished = false;
         bool m_processOnly = false;
+        AZStd::string m_filePath;
```

Comment:
I'm having a few bigger problems I could use help with... any thoughts on this?

```
<19:01:30> [Error] (Serialize) - Cannot enumerate data from data element (element:{E48A0B26-B6B7-4AF3-9341-9E5C5C1F0DE8}) from parent data element (element:{75F9EC0D-D8D6-5410-BD79-960E22076B03}) with class name "AZStd::unique_ptr<Contract>" because the ClassData does not exist. This can indicate that the class is not reflected at the point of this call. If this is class is reflected as part of a gem check if that gem is loaded
```
the gem is loaded and I saw `Contract` reflected, I even tried adding some `AZStd::unique_ptr<Contract>` to trigger on demand reflection but still not working...